dart-impl/base/CMakeLists.txt

```diff
@@ -120,6 +120,11 @@ set_target_properties(
   COMPILE_FLAGS ${ADDITIONAL_COMPILE_FLAGS}
 )
 
+set_target_properties(
+  ${DASH_DART_BASE_LIBRARY}
+  PROPERTIES POSITION_INDEPENDENT_CODE TRUE
+)
+
```

Comment:
Hmyes. I'd prefer a more canonical way myself to toggle this setting, like `BUILD_SHARED_LIBS`, but I'm not aware of any. Still, will wrap it in a build option `BUILD_PIC_LIBS` and disable it by default. It's sensible to assume that the common use case is to link DASH with an executable.